moped-editor/src/routes.js

```diff
@@ -7,14 +7,15 @@ import AccountView from "src/views/account/AccountView/AccountView";
 import StaffListView from "src/views/staff/StaffListView";
 import NewStaffView from "src/views/staff/NewStaffView";
 import EditStaffView from "src/views/staff/EditStaffView";
-import DashboardView from "src/views/reports/DashboardView/DashboardView";
+// import DashboardView from "src/views/reports/DashboardView/DashboardView";
```

Comment:
should we go ahead and just delete this?

Reply:
> On your question about how to style the links, I would just do the same blue text thats there for the project name. But kinda like the discussion was going, the way that works is kinda confusing from a user perspective.

+1 on link styling (but not a hill I'll die on if anyone else feels strongly) and agreed we should figure out a good pattern for editing those tabs while keeping the table in sight for the user. We can tackle that problem in subsequent issues/PRs for sure.

> Also, just from a bigger picture level, are we imagining updating the nav for this view? That's more of a product question.

@amenity and @patrickm02L can correct me here but I think we want "Dashboard" to stay and contain this signals table view even in the long-term, although the projects table that is generated will be filtered based on the user's affiliation with the project sponsor as more groups than AMD onboard. But overall we discussed the Dashboard being a combination of a few different views on the page (e.g. statistics, notifications, and other widgets may also be added to the Dashboard, as it would be the landing page for the user instead of the page containing all projects). Does that answer the question? (Sorry if I misunderstood!)